locale/de/LC_MESSAGES/django.po

```diff
@@ -986,6 +1038,12 @@ msgstr "Du hast Dich bereits am {date_time} für diese Schicht eingetragen."
 msgid "You successfully left this shift."
 msgstr "Du hast Deine Teilnahme abgesagt."
 
+msgid "You have no permissions to send emails!"
+msgstr "Du hast keine Erlaubnis um Emails zu senden!"
+
+msgid "Email has been sent."
+msgstr "Email wurde gesendet"
```

Comment:
Missing full stop at end of translation, while source language ends on full stop.